sdk/react/panels/videoView.js

```diff
@@ -272,6 +272,8 @@ class VideoView extends React.Component {
 
   _renderPlayPause = (show) => {
     const iconFontSize = ResponsiveDesignManager.makeResponsiveMultiplier(this.props.width, UI_SIZES.VIDEOVIEW_PLAYPAUSE);
+    const seekVisible = !this.props.config.live.forceDvrDisabled;
+    const notInLiveRegion = this.props.playhead <= this.props.duration * 0.95;
```

Comment:
what does 0.95 mean? Could you move it to constants?

Reply:
0.95 is a value from which player thinks that Live video should display Live icon, so we also disable FF button to synchronize them both.

>Could you move it to constants? 
Will do

Reply:
agree with @AlinaVoronkovaEpam , wherever you found the usage of 0.95 please move it all to a constant so that it's easier to understand.